### PR TITLE
fix: added function to get root path url in navigation service

### PR DESCRIPTION
**Pull request by Greedy-Geek**
## Description
Added get root path url in navigation service.

### Testing
Tested on the local

### Checklist:
-  My changes generate no new warnings

### CODE REVIEW

File: projects/common/src/navigation/navigation.service.ts [hunk 1]
```diff
@@ -272,6 +272,10 @@ export class NavigationService {
     return this.platformLocation.href;
   }
 
+  public getRootPathUrl(): string {
+    return window.location.host;
```
Comment (arjunlalb) [inline projects/common/src/navigation/navigation.service.ts:276]: please refer to L271

Comment (arjunlalb) [reply projects/common/src/navigation/navigation.service.ts:276]: what you need will be in `origin` in `platformLocation`

Comment (Greedy-Geek) [reply projects/common/src/navigation/navigation.service.ts:276]: platformLocation does not provide host, through that we'll need to add hostname and port, is that fine?

Comment (Greedy-Geek) [reply projects/common/src/navigation/navigation.service.ts]: updated

**Review by arjunlalb — APPROVED** [review]

**Pull request merged**